code/game/gamemodes/malfunction/malfunction.dm

```diff
@@ -267,8 +267,8 @@
 		world << "<B>The AI has chosen not to detonate the station!</B>"
 
 	else if (!station_captured &&                station_was_nuked)
-		feedback_set_details("round_end_result","halfwin - everyone killed by nuke")
-		world << "<FONT size = 3><B>Neutral Victory</B></FONT>"
+		feedback_set_details("round_end_result","draw - everyone killed by nuke")
+		world << "<FONT size = 3><B>Draw</B></FONT>"

```

Comment:
If the AI nukes the station, it gets a major victory. If the crew nukes the station, it's a draw?! Of course, the crew can only do that with admin intervention, but why is there a difference if the outcome is the same?